src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -56,6 +57,45 @@
 
 namespace EnergyPlus {
 
+    bool oneTimeBaseSizerFlag = true;
```

Comment:
There was a static one time flag in here that I extracted and pulled to this global level.